### ISSUE TITLE
Question about the stellar-hd-wallet

### ISSUE THREAD

**Issue by Ebioro** [chatch/stellar-hd-wallet#14]
Hi @chatch: Are there specific requirements to use the stellar-hd-wallet in an ionic-angular project?

I have tried a copule of things but none of them seems to work.

**Comment by chatch** [comment]
i've not used ionic-angular but it should work fine in most javascript projects. what's the problem?

**Comment by Ebioro** [comment]
Trying to do a small test of printing a mnemonic https://screencast.com/t/uDv2pEouRbSk. However, I get the following error: https://screencast.com/t/3CoZQGuT9ow.

**Comment by chatch** [comment]
this is not related to stellar-hd-wallet.  some problem in your recovery-phrase component there.

**Comment by Ebioro** [comment]
Yeah, that is what I thought first. But if I don't import the stellar-hd-wallet, the component renders just fine.  

I also tried to add the stellar-hd-wallet to the angular.json file like I did with the stellar-sdk https://screencast.com/t/nbkXEpXWU. Then I declare it like this https://screencast.com/t/T0ndoerFZh1
which gives me the following error https://screencast.com/t/J4L2f1kE6. Will keep trying options..
